src/proxy-ui-api/frontend/src/views/AddClient/SelectClientDialog.vue

```diff
@@ -0,0 +1,155 @@
+        <v-spacer />
+        <i @click="cancel()" data-test="x-close-button"></i>
+      </v-card-title>
+
+      <v-card-text style="height: 500px;" class="elevation-0">
+        <v-text-field
+          v-model="search"
+          :label="$t('wizard.client.member')"
+          single-line
+          hide-details
+          class="search-input"
+          data-test="client-search-input"
+        >
+          <v-icon slot="append">mdi-magnify</v-icon>
+        </v-text-field>
+
+        <!-- Table -->
+        <v-radio-group v-model="selectedMember">
+          <table class="xrd-table members-table fixed_header">
+            <thead>
+              <tr>
+                <th class="checkbox-column"></th>
+                <th>{{$t('name')}}</th>
+                <th>{{$t('localGroup.id')}}</th>
+              </tr>
+            </thead>
+            <tbody v-if="selectableClients && selectableClients.length > 0">
+              <tr v-for="member in filteredMembers()" v-bind:key="member.id">
+                <td class="checkbox-column">
+                  <div class="checkbox-wrap">
+                    <v-radio :key="member.id" :value="member"></v-radio>
+                  </div>
+                </td>
+
+                <td>{{member.member_name}}</td>
+                <td>{{member.id}}</td>
+              </tr>
+            </tbody>
+          </table>
+        </v-radio-group>
+
+        <div v-if="filteredMembers().length < 1" class="empty-row">
+          <p>{{$t('localGroup.noResults')}}</p>
+        </div>
+      </v-card-text>
+      <v-card-actions class="xrd-card-actions">
+        <v-spacer></v-spacer>
+
+        <large-button
+          class="button-margin"
+          outlined
+          @click="cancel()"
+          data-test="cancel-button"
+        >{{$t('action.cancel')}}</large-button>
+
+        <large-button
+          :disabled="!selectedMember"
+          @click="save()"
+          data-test="save-button"
+        >{{$t('localGroup.addSelected')}}</large-button>
+      </v-card-actions>
+    </v-card>
+  </v-dialog>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { mapGetters } from 'vuex';
+import LargeButton from '@/components/ui/LargeButton.vue';
+import { Client } from '@/types';
+
+export default Vue.extend({
+  components: {
+    LargeButton,
+  },
+  props: {
+    dialog: {
+      type: Boolean,
+      required: true,
+    },
+    selectableClients: {
+      type: Array,
+      required: true,
+    },
+  },
+
+  data() {
+    return {
+      search: '',
+      selectedMember: undefined,
+    };
+  },
+  methods: {
+    filteredMembers() {
+      if (!this.search) {
+        return this.selectableClients;
+      }
+
+      const tempSearch = this.search
+        .toString()
+        .toLowerCase()
+        .trim();
+      if (tempSearch === '') {
+        return this.selectableClients;
+      }
+
+      return this.selectableClients.filter((member: any) => {
+        if (
```

Comment:
optional chaining here and next if condition

Reply:
Fixed